src/main/scala/gitbucket/core/controller/AccountController.scala

```diff
@@ -150,20 +150,21 @@ trait AccountControllerBase extends AccountManagementControllerBase {
 
   get("/:userName/_avatar"){
     val userName = params("userName")
+    contentType = "image/png"
     getAccountByUserName(userName).map{ account =>
       response.setDateHeader("Last-Modified", account.updatedDate.getTime)
       account.image.map{ image =>
         RawData(FileUtil.getMimeType(image), new java.io.File(getUserUploadDir(userName), image))
       }.getOrElse{
-        contentType = "image/png"
         (if (account.isGroupAccount) {
           TextAvatarUtil.textGroupAvatar(account.fullName)
         } else {
           TextAvatarUtil.textAvatar(account.fullName)
-        }).getOrElse(Thread.currentThread.getContextClassLoader.getResourceAsStream("noimage.png"))
+        }).get
```

Comment:
Don't use `Option.get` if there are any possibility to `None` is got here. If `TextAvatarUtil` can always return an image, it shouldn't be `Option`.
I think that to fallback to `noimage.png` is good. Why did you remove that?